fabtests/common/shared.c

```diff
@@ -3052,13 +3052,13 @@ void ft_fill_buf(void *buf, int size)
 	}
 }
 
-int ft_check_buf(void *buf, int size)
+int ft_check_buf(void *buf, size_t size)
 {
 	char *recv_data;
 	char c;
 	static unsigned int iter = 0;
 	int msg_index;
-	int i;
+	size_t i;
```

Comment:
I think we also need to change [L3074](https://github.com/ofiwg/libfabric/pull/6229/commits/c8c0b21027d57ac53ce03c0e2ef16e32865dab51#diff-e6321b146969c833bd5a352ffbcbb8baR3074) to %ld for size and i. Otherwise we will get compilation warning
```
common/shared.c:3074:39: warning: format '%d' expects argument of type 'int', but argument 3 has type 'size_t {aka long unsigned int}' [-Wformat=]
   printf("Error at iteration=%d size=%d byte=%d\n",
                                      ~^
                                      %ld
```

Reply:
Good call. Updated.